app/controllers/static_controller.rb

```diff
@@ -0,0 +1,8 @@
+class StaticController < ApplicationController
```

Comment:
instead of this controller we can add a custom method to the registrations controller which would make sense since all that is related to registration anyway?